smokey/src/main/java/frc/robot/commands/AllignWithTag.java

```diff
@@ -0,0 +1,83 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: AllignWithTag.java
+// Intent: Forms a command to allign itself with a designated april tag.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.math.controller.PIDController;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.common.MotorUtils;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class AllignWithTag extends CommandBase{
+    private boolean done = false;
+    private double tagID;
+    private double velocityValue = 0.3;
```

Comment:
If I told you that this should be marked with the 'final' keyword, would you be able to tell me why?  In addition to adding the modifier, tell me why final should be associated with 'velocityValue'.

Reply:
because it never changes. 
and added final.